### DISCUSSION TITLE
Unit for kinetic parameters?

### DISCUSSION THREAD

**Discussion by ogandril** [theislab/scvelo#478] [Q&A]
Just to be on the safe side: can you confirm that all rates of RNA transcription, splicing and degradation are expressed in hours^-1? (i.e. 1/h)?

**Comment by WeilerP** [comment]
From a biological standpoint it is reasonable to assume the rates to be measure per hour. From a mathematical point of view, however, there is no such restriction. The unit of measured data is counts everything else has to be inferred. If we were able to measure the velocity as well, the unit for the model parameters would be given as well.
